### DISCUSSION TITLE
Grove - Analog Microphone | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#184] [Q&A]
# Grove - Analog Microphone | Seeed Studio Wiki

The Grove - Analog Microphone is a based on high-performance SiSonic MEMS technology, offering an extremely-low-noise, low-current, reliable, and small microphone to opensource hardware industry, and it has improved performance under severe conditions.

https://wiki.seeedstudio.com/Grove-Analog-Microphone/

**Comment by f-l-s** [comment]
There are examples of usage but no technical descriptIon of the device, what do we have in output (white / yellow)?
A mono signal ? 
It’s a bit surprising to not have a clear description of this sensor.
Hope you will add it.
Thank in advance.

**Comment by limengdu** [comment]
Thank you for your feedback, we will take time to improve our Wiki afterwards. until then, yellow is mic_p and white is mic_n, hope there is no delay in your use.
